instant-clip-tokenizer-py/src/lib.rs

```diff
@@ -25,6 +25,9 @@ impl Tokenizer {
     ///
     /// The resulting `Tokenizer` is suitable for use with the original CLIP
     /// model.
+    ///
+    /// Note that creating a new `Tokenizer` tends to be somewhat expensive, so
```

Comment:
I think we should omit the word `somewhat` here. It either is expensive (enough to worry about) or it isn't -- see also [weasel words](https://proofed.com/writing-tips/what-are-weasel-words/).

Reply:
Will do that, thanks for the suggestion and the link about weasel words - very interesting read indeed! :) I also changed the doc comments of all other constructors.